cvat-core/.eslintrc.js

```diff
@@ -4,6 +4,7 @@
 
 module.exports = {
     env: {
+        amd: true,
```

Comment:
Why do you need this change?

Reply:
Due to require warining

Reply:
Set ``node`` (row below) to true instead